app/controllers/users_controller.rb

```diff
@@ -0,0 +1,70 @@
+class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
+  before_action :correct_user,   only: [:show, :edit, :update]
+  before_action :admin_user,     only: :destroy
```

Comment:
`check_admin` or `authorize_admin`

Reply:
I understand some conventions are good practice, but this came straight from the book, and i did not get at the admin namespace since i was waiting on finishing the cart & user orders before getting to it. Will get on it tho.